lib/bundlex/platform.ex

```diff
@@ -34,14 +34,29 @@ defmodule Bundlex.Platform do
   end
 
   @doc """
-  Detects current platform.
+  Detects target platform.
 
   In case of success returns platform name
 
   Otherwise raises Mix error.
   """
-  @spec get_current! :: name_t
-  def get_current! do
+  @spec get_target!() :: name_t
+  def get_target!() do
+    case {Mix.target(), System.fetch_env("NERVES_APP")} do
```

Comment:
Looks like this `Mix.target()` call breaks Elixir releases, as Mix functions are not available in runtime and this apparently is called on startup.
```
Kernel pid terminated (application_controller) ({application_start_failure,kernel,{{shutdown,{failed_to_start_child,on_load,{on_load_function_failed,'Elixir.Membrane.RTMP.Sink.Native.Nif',{undef,[{'Elixir.Mix',target,[],[]},{'Elixir.Bundlex.Platform','get_target!',0,[{file,"lib/bundlex/platform.ex"},{line,45}]},{'Elixir.Bundlex',family,0,[{file,"lib/bundlex.ex"},{line,51}]},{'Elixir.Bundlex.Helper.PathHelper',fix_slashes,1,[{file,"lib/bundlex/helper/path_helper.ex"},{line,25}]},{'Elixir.Membrane.RTMP.Sink.Native.Nif',load_nif,0,[{file,"lib/membrane_rtmp_plugin/rtmp/sink/native.ex"},{line,1}]},{init,'-run_on_load_handler/2-fun-0-',1,[]}]}}}},{kernel,start,[normal,[]]}}})
```